src/main/java/cl/transbank/webpay/webpayplus/WebpayPlus.java

```diff
@@ -54,16 +69,42 @@ public static void configureDeferredForTesting() {
                 "579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C", IntegrationType.TEST));
     }
 
-    public static class Transaction {
+    private static Options buildNormalOptions(Options options) {
+        // set default options for Webpay Plus Normal if options are not configured yet
+        if (WebpayPlus.getOptions().isEmpty())
+            configureNormalForTesting();
+
+        return WebpayPlus.getOptions().buildOptions(options);
+    }
+
+    public static class Transaction extends WebpayApiResource {
         public static CreateWebpayPlusTransactionResponse create(
                 String buyOrder, String sessionId, double amount, String returnUrl) throws TransactionCreateException {
-            return null;
+            return create(buyOrder, sessionId, amount, returnUrl, null);
         }
 
         public static CreateWebpayPlusTransactionResponse create(
                 String buyOrder, String sessionId, double amount, String returnUrl, Options options)
                 throws TransactionCreateException {
-            return null;
+            try {
+                options = buildNormalOptions(options);
+
+                final URL endpoint = new URL(getCurrentIntegrationTypeUrl(options.getIntegrationType()));
+
+                final TransactionCreateRequest in = new TransactionCreateRequest(buyOrder, sessionId, amount, returnUrl);
+                final TransactionCreateResponse out = getHttpUtil().request(endpoint, HttpUtil.RequestMethod.POST, in,
+                        buildHeaders(options), TransactionCreateResponse.class);
+
+                if (null == out)
+                    throw new TransactionCreateException("No fue posible obtener respuesta desde el servicio de transbank");
```

Comment:
Is it correct to have error messages in Spanish?

Reply:
Fixed